test/python/tensorflow/python_tests_list_gpu.txt

```diff
@@ -389,6 +389,73 @@ relu_op_test.ReluTest.testGradientFloat32
 relu_op_test.ReluTest.testGradientScalar
 relu_op_test.ReluTest.testNumbers
 
+scatter_nd_ops_test.ScatterNdNonAliasingAddTest.testBool
```

Comment:
Have these been run on GPU?

Reply:
I thought of removing them (since they are removed on CPU), but kept them there since there is no CI for GPU. If/when GPU ci is run, I suspect some other tests will fail too, and we can comment them out then.

For mac, CI will run now, and if it fails (which I think it will) I'll remove it.

Reply:
Removed from mac as they fail there